models/FESOM/README.rst

```diff
@@ -0,0 +1,390 @@
+|             | fesom_dart_mod.F90                   | DART diagnostic  |
+|             |                                      | output routines  |
++-------------+--------------------------------------+------------------+
+|             | fesom_forcing_mod.F90                | forcing          |
+|             |                                      | diagnostic       |
+|             |                                      | routines         |
++-------------+--------------------------------------+------------------+
+|             | fesom_observation_mod.F90            | observation      |
+|             |                                      | diagnostic       |
+|             |                                      | routines         |
++-------------+--------------------------------------+------------------+
+|             | gen_input.F90                        | routines for     |
+|             |                                      | I/O (adapted     |
+|             |                                      | from FESOM)      |
++-------------+--------------------------------------+------------------+
+|             | gen_modules_clock.F90                | routines for     |
+|             |                                      | timing (adapted  |
+|             |                                      | from FESOM)      |
++-------------+--------------------------------------+------------------+
+|             | gen_modules_config.F90               | routines for     |
+|             |                                      | configuration    |
+|             |                                      | (adapted from    |
+|             |                                      | FESOM)           |
++-------------+--------------------------------------+------------------+
+|             | mesh_read.F90                        | routines for     |
+|             |                                      | reading the      |
+|             |                                      | mesh (adapted    |
+|             |                                      | from FESOM)      |
++-------------+--------------------------------------+------------------+
+|             | Makefile                             | Makefile         |
+|             |                                      | (adapted from    |
+|             |                                      | FESOM) but       |
+|             |                                      | reads DART       |
+|             |                                      | environment      |
++-------------+--------------------------------------+------------------+
+|             | oce_dens_press.F90                   | routines to      |
+|             |                                      | compute density  |
+|             |                                      | and              |
+|             |                                      | p                |
+|             |                                      | ressure(adapted  |
```

Comment:
p and ressure should be connected.  This happens a bit throughout the document.

Reply:
Fixed.